web/src/components/shared/cart/cart-purchase-summary.tsx

```diff
@@ -42,6 +51,28 @@ const Total = styled.div`
 `;
 
 const PurchaseSummary = ({ cart }: { cart: CP[] }) => {
+  const [Error, setError] = useState('');
+  const clickHandler = () => {
```

Comment:
I found a bug related to the cart and the navbar, sometimes when you are not at the top of the page, the navbar is displayed(because you scrolled up) and then you click the cart, then the navbar closes pulling the cart up making it impossible to close the cart and leaving a white gap in the screen 
<img width="1680" alt="Screenshot 2020-10-03 at 10 48 14" src="https://user-images.githubusercontent.com/8405459/94987355-33469980-0566-11eb-91f3-694a792f690a.png">